sdk/storage/src/blobs/blob_service_client.cpp

```diff
@@ -159,4 +159,20 @@ namespace Azure { namespace Storage { namespace Blobs {
         options.Context, *m_pipeline, m_serviceUrl.ToString(), protocolLayerOptions);
   }
 
+  Azure::Core::Response<AccountInfo> BlobServiceClient::GetAccountInfo(
+      const GetAccountInfoOptions& options) const
+  {
+    BlobRestClient::Service::GetAccountInfoOptions protocolLayerOptions;
+    return BlobRestClient::Service::GetAccountInfo(
+        options.Context, *m_pipeline, m_serviceUrl.ToString(), protocolLayerOptions);
+  }
+
+  Azure::Core::Response<BlobServiceStatistics> BlobServiceClient::GetStatistics(
```

Comment:
Please don't provide API that does not work right now.

Reply:
removed